frontend/model/contracts/group.js

```diff
@@ -642,6 +643,14 @@ sbp('chelonia/defineContract', {
         archiveProposal({ state, proposalHash: data.proposalHash, proposal, contractID })
       }
     },
+    'gi.contracts/group/notifyExpiringProposals': {
+      validate: arrayOf(string),
+      process ({ data, meta, contractID }, { state }) {
+        for (const proposalId of data) {
+          Vue.set(state.proposals[proposalId], 'notifiedBeforExpire', true)
```

Comment:
Typo: `notifiedBeforExpire` => `notifiedBeforeExpire`

Remember to update that everywhere it's used, and also remember to re-run `grunt pin` with `--overwrite`